components/micro-gateway-cli/src/main/resources/templates/policy.mustache

```diff
@@ -45,7 +45,7 @@ forever {
     }
 
     from s{{name}}resultStream
-    window gateway:emitOnStateChange(s{{name}}resultStream.throttleKey, s{{name}}resultStream.isThrottled, "s{{name}}resultStream")
+    throttler:emitOnStateChange(s{{name}}resultStream.throttleKey, s{{name}}resultStream.isThrottled)
```

Comment:
Lets use the old namespace

Reply:
due to an issue in new ballerina this namespace cannot be used